tests/test_filtsmooth/test_gaussfiltsmooth/test_kalman_utils.py

```diff
@@ -0,0 +1,125 @@
+"""Test Kalman utility functions."""
+
+
+import numpy as np
+import pytest
+
+import probnum.filtsmooth as pnfs
+import probnum.filtsmooth.statespace as pnfss
+import probnum.random_variables as pnrv
+from probnum.problems.zoo.linalg import random_spd_matrix
+
+
+@pytest.fixture
+def ordint():
+    return 0
+
+
+@pytest.fixture
+def ordint():
+    return 1
+
+
+@pytest.fixture
+def ordint():
+    return 6
+
+
+@pytest.fixture
+def spatialdim():
+    return 1
+
+
+@pytest.fixture
+def spatialdim():
```

Comment:
I have 0 pytest experience, but the above fixtures feel a bit weird. I would expect there might be an easier way to achieve that? I saw [here](https://docs.pytest.org/en/stable/parametrize.html#parametrize) that `@pytest.fixture` has a `params` kwarg, and there is also a `@pytest.mark.parametrize`.

But again, 0 pytest experience here, so I'm 100% fine if you want to leave it as it is! :)

Reply:
I am not sure either here... I think one could also parametrise the test functions below, but wouldnt we need to copy the string `pytest.mark.parametrize(["ordint", "spatialdim"], [[0, 1, 6], [1, 4]]` to all the test functions that need this input?

Reply:
That might be. Since we don't know, maybe we just keep it like that and just refactor later if necessary?

Reply:
Okay! I suspect that by the time our pytest skills improve, we may come back to the tests anyway.